conference/submissions.md

```diff
@@ -23,8 +24,8 @@ would be glad to have hybrid online/in person events proposed).
 ### Timeline and process
 
 Submission opens at the end of July and is planned to close on
-9.september.  Submissions should consist of ~150 word abstract for
-shorter talks, and ~250 words for longer talks.  There will be a
+September 9.  Submissions should consist of ~150 word abstract for
+shorter talks, and ~250 words for longer talks.  There will be an anonymized
```

Comment:
There's blind and anonymous.  Would we want to do blind, but identities of reviewers become known afterwards?

Identities of people with accepted talks clearly become known by conference time...

Reply:
Clearly a question for PC... if there is time to ask?

Reply:
Not sure what the right term is but I think identities of submitters are not known to reviewers but also reviewers are not revealed. Only one (or few) person(s) have the table matching submitter and reviewer names to numbers/codes. This one/few persons would not take part in the actual review.

Reply:
I will add a commit that makes this more explicit.

Reply:
Are the reviewers' identities revealed *after* the review phase?  I've seen this in at least one journal I reviewed for, and just saw that JupyterCon does this.  Regardless, we should definitely do blind *during* the review phase.

Random idea: "proposal incubator" - people can get feedback on their talks and ideas before the review process.  That could help to nurture a new audience who isn't sure if they fit in.

Reply:
Added a commit. I like the proposal incubator. Will spawn an issue in a minute.